Bio/PDB/PDBList.py

```diff
@@ -479,17 +501,19 @@ def get_all_assemblies(self, file_format="mmCif"):
 
         # FTPLib is much faster than urlopen
```

Comment:
This is a shame really. FTPLib is really much faster for some tasks.. shame that they'll drop the protocol.

Reply:
In fact, it's debatable, but do not worry about the speed. Implementing parallel download will speed up the process so much that it will no longer be an issue (but that is another step!).